src/field/password.js

```diff
@@ -1,8 +1,11 @@
-import createPolicy from 'password-sheriff';
+import PasswordPolicy from 'password-sheriff/lib/policy';
 import { setField } from './index';
 
 export function validatePassword(password, policy) {
-  return createPolicy(policy).check(password);
+  if (!policy) {
+    return true;
+  }
+  return new PasswordPolicy(policy.toJS()).check(password);
```

Comment:
Shouldn't this be safer, e.g. checking that if `policy == undefined` then `none` policy is still applied? https://github.com/auth0/password-sheriff/blob/master/index.js#L57 Now it will work since `none` is just `length > 0` but if that changes this code will need to be changed as well. Let's reference this to the `none` policy instead.

Reply:
policy is only undefined when it's a login, which we don't validate passwords.